slides.md

```diff
@@ -680,6 +680,9 @@ BONUS STEP.
 
 # Next Steps
 
+- Create a **Composition** to expose your backend (see [here](https://blog.platformatic.dev/introducing-platformatic-composition))
+- Deploy all your services as a **Runtime** (see [here](https://blog.platformatic.dev/introducing-platformatic-runtime))
```

Comment:
Well, is "next steps", so IMO something you can do as further thing.
In both cases, though, you need more services (which are not defined), because now we have just one Platformatic DB service, accessed directly.   
Will clarify that. Or, if it's confusing, I can remove this. WDYT @mcollina ?

Reply:
Removed.